src/http/server/response.cr

```diff
@@ -79,6 +79,15 @@ class HTTP::Server
       raise "Can't read from HTTP::Server::Response"
     end
 
+    # Send a 100 Continue response to clients waiting for this before sending
+    # the request body
+    def continue
+      @status_code = 100
```

Comment:
I'd suggest to let the status code be configurable as argument (default to `100`). You might want to use this mechanic with other status codes (`102` for example).

Reply:
Alternatively, only set it to 100 if it was not explicitly set before, but that'd be kind of hart to detect.

Reply:
sure, why not. Should it still be called continue then?

Reply:
Good question. I'm not sure about the name anyway.
Maybe something like `send_intermediate_headers`?

Reply:
Maybe we should do some research how other web frameworks/servers call this...